orgs/trial.json

```diff
@@ -1,6 +1,6 @@
 {
   "orgName": "NPSP - Trial Org",
-  "template": "0TT1I000006q2o6",
+  "template": "0TT1I000006q2sd",
```

Comment:
I'm not sure this change should be in here

Reply:
It was a file changed in `master`. After merging `feature/RD2__rad-auto-populate-rd-name` into this branch, file differences show only modifications done for this WI.